### DISCUSSION TITLE
IntegrateData - Finding integration vector weights Error in Embeddings(reduction)[nn.cells2, dims] : subscript out of bounds

### DISCUSSION THREAD

**Discussion by ZhenyaoYe** [satijalab/seurat#7489] [Q&A]
Hi, 

Would there be someone available who could kindly assist me with the issue I encountered while utilizing the IntegrateData function?

Finding integration vectors
Warning in irlba(A = t(x = object), nv = npcs, ...) :
  You're computing too large a percentage of total singular values, use a standard svd instead.
Warning in irlba(A = t(x = object), nv = npcs, ...) :
  did not converge--results might be invalid!; try increasing work or maxit
**Finding integration vector weights
Error in Embeddings(reduction)[nn.cells2, dims] : subscript out of bounds**

Thank you! 

Best regards,
Zhenyao

**Comment by AlonsoOga** [comment]
I'm encountering a similar issue related to vector weights

`Warning in irlba(A = t(x = object), nv = npcs, ...) :
  did not converge--results might be invalid!; try increasing work or maxit
Finding integration vector weights
Error in idx[i, ] <- res[[i]][[1]] : 
  number of items to replace is not a multiple of replacement length`
